src/lib/autoinstall/widgets/storage/mkfs_options.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "cwm/common_widgets"
+
+module Y2Autoinstallation
+  module Widgets
+    module Storage
+      # Widget to specify the mkfs command options
+      class MkfsOptions < CWM::InputField
+        def initalize
+          textdomain "autoinst"
+          super
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          _("Mkfs options")
```

Comment:
Label capitalization again

Reply:
Sorry, I competely forgot to apply the [Title Capitalization Rules in English](https://en.wikipedia.org/wiki/Capitalization_in_English#Title_capitalization_in_different_styles) in the whole PR.

I am fixing it. 

Thanks.